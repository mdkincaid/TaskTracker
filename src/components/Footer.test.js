// react
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

// project
import Footer from './Footer';

describe('Footer', () => {
	it('should render', () => {
		const result = render(
			<Router>
				<Footer />
			</Router>
		);

		// assert
		expect(result.container).toMatchSnapshot();
	});
});
