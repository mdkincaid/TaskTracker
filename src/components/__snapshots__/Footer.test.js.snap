// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<div>
  <footer>
    <p>
      Copyright Â© 2021
    </p>
    <a
      href="/about"
    >
      About
    </a>
  </footer>
</div>
`;
